ethcore/builtin/src/lib.rs

```diff
@@ -245,6 +245,13 @@ impl Builtin {
 	pub fn is_active(&self, at: u64) -> bool {
 		self.pricer.range(0..=at).last().is_some()
 	}
+
+	/// EIP 1352: Restricted address range for precompiles/system contracts
+	#[inline]
+	pub fn in_precompiles_address_range(addr : &Address) -> bool {
+		let limit = Address::from_low_u64_ne(0xffff);
+		addr.cmp(&limit) !=  Ordering::Greater
```

Comment:
is it a more obscure way of writing `addr <= &limit`?

Reply:
fixed in b34b753